evap/staff/templates/staff_course_email.html

```diff
@@ -2,18 +2,52 @@
 
 {% load i18n %}
 {% load static %}
+{% load morefilters %}
 
 {% block content %}
     {{ block.super }}
+    {% if form.errors %}
+        <div class="alert alert-danger" role="alert">{% blocktrans %}Please fill out all required fields and select at least one recipient group.{% endblocktrans %}</div>
+    {% endif %}
```

Comment:
is this necessary? i mean the form is small enough and the errors of the individual fields should be explicit enough, or aren't they?